pajbot/modules/chat_alerts/livealert.py

```diff
@@ -44,9 +44,9 @@ def on_stream_start(self, **rest):
         streamer = self.bot.streamer_display
         game = self.bot.stream_manager.game
         title = self.bot.stream_manager.title
-        self.bot.say(live_chat_message.format(streamer=streamer, game=game, title=title))
+        self.bot.safe_say(live_chat_message.format(streamer=streamer, game=game, title=title))
         if self.settings["extra_message"] != "":
-            self.bot.say(self.settings["extra_message"].format(streamer=streamer))
+            self.bot.safe_say(self.settings["extra_message"].format(streamer=streamer))
```

Comment:
No need to run safe_say on this as it's triggered by a broadcaster and moderators actions.
Moderator sets the message, streamer sets the game/title.